statsproxy.go

```diff
@@ -47,134 +33,50 @@ func monitor() {
 		defer pprof.StopCPUProfile()
 	}
 
-	ticker := time.NewTicker(time.Duration(config.Service.TickerPeriod) * time.Second)
 signalLoop:
 	for {
 		select {
 		case sig := <-signalchan:
 			common.Logger.Info(fmt.Sprintf("!! Caught signal %d... shutting down\n", sig))
 			break signalLoop
-		case <-ticker.C:
-			sendStats()
+		case <-common.StatsTicker.C:
+			workers.SendStats()
 		}
 	}
 }
 
-func udpListener(listener *net.UDPConn) {
-	message := make([]byte, config.Service.MaxUDPPacketSize)
-	for {
-		n, remaddr, err := listener.ReadFromUDP(message)
-		if err != nil {
-			common.Logger.Err(fmt.Sprintf("ERROR: reading UDP packet from %+v - %s",
-				remaddr, err))
-			continue
-		}
-
-		dataHandler(message[:n])
+func initializeUDPListener() *net.UDPConn {
+	address, er := net.ResolveUDPAddr("udp", config.Service.Port)
```

Comment:
bikesheddy, but it's more idomatic for `er` to be called `err`.

Please don't hurt me :no_bicycles: :